prow/tide/tide.go

```diff
@@ -955,9 +955,9 @@ func (c *Controller) pickBatch(sp subpool, cc map[int]contextChecker) ([]PullReq
 
 	var candidates []PullRequest
 	for _, pr := range sp.prs {
-		if isPassingTests(sp.log, c.ghc, pr, cc[int(pr.Number)]) {
-			candidates = append(candidates, pr)
-		}
+		//if isPassingTests(sp.log, c.ghc, pr, cc[int(pr.Number)]) {
```

Comment:
let's throw a comment in here explaining the reasoning behind doing this

Reply:
yup :+1:

Reply:
going to add an argument to control this behavior as well